specs/phase0/beacon-chain.md

```diff
@@ -1384,20 +1431,57 @@ def get_attestation_deltas(state: BeaconState) -> Tuple[Sequence[Gwei], Sequence
         rewards[attestation.proposer_index] += proposer_reward
         max_attester_reward = get_base_reward(state, index) - proposer_reward
         rewards[index] += Gwei(max_attester_reward // attestation.inclusion_delay)
+    return rewards, penalties
```

Comment:
to make it clear that `penalties ` are all zeros, maybe initialize `penalties ` before this line.
```python
    penalties = [Gwei(0) for _ in range(len(state.validators))]
    return rewards, penalties
```